Python/pywarpx/picmi.py

```diff
@@ -525,7 +525,6 @@ class ElectromagneticSolver(picmistandard.PICMI_ElectromagneticSolver):
     def init(self, kw):
         assert self.method is None or self.method in ['Yee', 'CKC', 'PSATD'], Exception("Only 'Yee', 'CKC', and 'PSATD' are supported")
 
-        self.do_pml = kw.pop('warpx_do_pml', None)
         self.pml_ncell = kw.pop('warpx_pml_ncell', None)
```

Comment:
@dpgrote independent of this PR, should we throw a warning in PICMI `init` methods about unused `kw` keywords?
I think we would silently ignore additional arguments here, which can be hard to debug for users.

Reply:
This is already done, in the pimcistandard code. It happens at line 47 in the file `picmi/PICMI_Python/base.py`.

Reply:
Ah, very cool!
https://github.com/picmi-standard/picmi/blob/4bb9858dd6b3c2e80b4e06c62a11a53b3ce9a70e/PICMI_Python/base.py#L47-L48